cluster/addons/kube-proxy/kube-proxy-ds.yaml

```diff
@@ -48,7 +48,8 @@ spec:
         {{kube_cache_mutation_detector_env_name}}
```

Comment:
note the comment at the top about keeping this in sync with kube-proxy.manifest

Reply:
Thanks for the tip.  Unfortunately I am not seeing the file to which that is referring.

Reply:
Should be here https://github.com/kubernetes/kubernetes/blob/master/cluster/gce/manifests/kube-proxy.manifest